docs/development/index.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: default
+title: Ontwikkeling van een integratie
+has_children: false
+has_toc: true
+nav_order: 3
+---
+
+<details markdown="block">
+  <summary>
+    Inhoudstafel
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+# Ontwikkeling van een integratie
+
+## Introductie
+
+Content partners kunnen zelf aan de slag met onze API's en services om een
+integratie te bouwen. Hieronder vind je informatie om hier mee te kunnen
+starten.
+
+## De verschillende omgevingen en hun endpoints
+
+Meemoo heeft twee - publieke - omgevingen:
+
+- QAS: of "staging", een omgeving die productie nabootst en waarin
+  ontwikkelaars en CP's kunnen testen.
+- PRD: of "production", de werkelijke omgeving.
+
+De gewoonlijke (en **sterk** aangeraden) procedure is om eerst in de
+QAS-omgeving te testen en pas als alles naar behoren werkt en uitvoerig is
+getest, naar productie te gaan.
+
+### Netwerk locaties
+
+Hieronder vind je de verschillende netwerk locaties eigen aan elke omgeving.
+
+| QAS                 | PRD             |
+|---------------------|-----------------|
+| archief-qas.viaa.be | archief.viaa.be |
+
+**Note**: de transitie naar het meemoo-domein is nog niet volledig afgerond.
+Voorlopig is het beter en veiliger om nog het viaa-domein te gebruiken. Sowieso
+is het goed om bovenstaande netwerk locaties als omgevingsvariabelen
+("_environment variables_") te configureren in je applicatie of integratie.
+
+### Netwerk paden
```

Comment:
I'd mention this later, because the different APIs haven't been introduced.

Reply:
Good point. We'll remember this for the next iteration.